templates/tfengine/components/cicd/manual/main.tf

```diff
@@ -54,12 +54,15 @@ locals {
   cloudbuild_sa_viewer_roles = [
     "roles/browser",
     "roles/iam.securityReviewer",
+    "roles/secretmanager.viewer",
```

Comment:
why do we need these additional permissions?

Reply:
Doesn't the CICD need these permissions to access / deploy secrets?

Reply:
But the SA already has owner role to each project though. And we don't want users to put secrets in the devops project, so shouldn't grant SA secret admin role in devops project.

Reply:
Oh right.. good point. I'll remove this.